### ISSUE TITLE
[Bug] All items can be crafted into charcoal

### ISSUE THREAD

**Issue by MrShellad** [quiqueck/BetterEnd#41]
### What happened?

All items can be crafted into charcoal

### BetterEnd

2.0.9

### BCLib

2.0.15

### Fabric API

qfapi-2.0.0-beta.6_qsl-2.0.0-beta.14_fapi-0.57.0

### Fabric Loader

quilt

### Minecraft

1.19

### Other Mods

```shell
AdditionalAdditions-5.0.1.jar
AdditionalStructures-1.19-(v.4.0.0).jar
AdvancementPlaques-1.19-fabric-1.4.5.jar
adventurez-1.4.14.jar
AmbientSounds_FABRIC_v5.2.0_mc1.19.jar
animal_feeding_trough-1.0.3+1.19-quilt (1).jar
animatica-0.5+1.19.jar
appleskin-fabric-mc1.19-2.4.0.jar
architectury-5.9.31-fabric.jar
armorpointspp-fabric-1.19-3.0.5.jar
artifacts-7.1.1+fabric.jar
AttributeFix-Fabric-1.19-15.0.2.jar
AwesomeDungeonOcean-Fabric-1.19-2.0.8.jar
balm-fabric-4.3.0+0.jar
bclib-2.0.15.jar
BeaconOverhaul-1.7.0+1.19.jar
better-end-2.0.9.jar
better-nether-7.0.6.jar
BetterAdvancements-Fabric-1.19-0.2.1.134.jar
betteranimalsplus-1.19-11.0.6-fabric.jar
betterfpsdist-fabric-1.19-1.9.jar
BetterThirdPerson-Fabric-1.19-1.8.1.jar
biomemakeover-FABRIC-1.19-1.5.8.jar
Bookshelf-Fabric-1.19-14.1.7.jar
borderless-mining-1.1.5+1.19.jar
Bridging-1.19-1.1.0.jar
cardinal-components-api-5.0.1.jar
cave-enhancements-1.2.8.jar
cem-0.7.1.jar
charm-fabric-1.19-4.1.2.jar
charmonium-fabric-1.19-4.1.0.jar
chat_heads-0.6.0-fabric-1.19.jar
Chunky-1.2.217.jar
cleancut-1.19-5.0-fabric.jar
cloth-api-4.0.65.jar
cloth-config-7.0.73-fabric.jar
Clumps-fabric-1.19-9.0.0+3.jar
collective-fabric-1.19.x-4.27.jar
colormatic-3.1.2+mc.1.19.jar
continuity-2.0.0+1.19.jar
Controlling-fabric-1.19-10.0+3.jar
coppermod-1.19.jar
CosmeticArmor-1.19-1.4.0.jar
cozycabincore-1.1-1.19.jar
CreativeCore_FABRIC_v2.7.2_mc1.19.jar
crimson-moon-2.4.0-1.19.jar
custom-spawns-0.7+1.19.jar
DarkPaintings-Fabric-1.19-11.0.2.jar
diggusmaximus-1.5.6-1.19.jar
disenchanter-0.1.7+1.19.jar
dragonloot-1.1.1.jar
dusk-fabric-1.19-1.8.jar
DynamicSoundFilters-1.3.0+1.19.jar
earthbounds-1.19-1.1.3.jar
earthtojavamobs-fabric-mc1.19-1.9.2+1.19.jar
ecologics-fabric-1.19-2.1.4.jar
EnchantmentDescriptions-Fabric-1.19-11.0.5.jar
enchantwithmob-quilt-1.1.0+1.19.jar
enderscape-0.3.4-1.19.jar
enhancedblockentities-0.7.1+1.19.jar
entity-texture-features-3.1.4-mc1.19.jar
entityculling-fabric-1.5.2-mc1.19.jar
EquipmentCompare-1.19-fabric-1.3.1.jar
extraalchemy-fabric-1.19-1.8.3.jar
extrasounds-2.3.1+1.19.jar
fabric-end-remastered-5.1.0-1.19.jar
fabric-language-kotlin-1.8.1+kotlin.1.7.0.jar
fabric-seasons-1.4-BETA+1.19.jar
fabrictailor-1.8.4.jar
fallingleaves-1.12.1+1.19.jar
FallingTree-1.19-3.7.4.jar
farmers-delight-fabric-1.19-0.3.0.jar
farmersknives-2.0.jar
farsighted-mobs-fabric-1.1-1.18-1.19.jar
ForgeConfigAPIPort-v4.0.0-1.19-Fabric.jar
FpsReducer2-fabric-1.19-2.0.2.jar
frame-fabric-0.22.2+1.19.jar
friendsandfoes-fabric-mc1.19-1.5.5.jar
geckolib-fabric-1.19-3.1.5.jar
goblin-traders-1.5.2.jar
graves-2.1.0+1.19.jar
guard-villagers-quilt-1.19-1.0.15 (1).jar
hallownestexpansion-0.1.1-1.19.jar
handoveryouritems-fabric_1.19.x-1.5.jar
Highlighter-1.19-fabric-1.1.2.jar
Iceberg-1.19-fabric-1.0.38.jar
Illager Plushies FABRIC 1.19.jar
illuminations-1.10.8.jar
imblockerfabric-1.0.19.jar
indium-1.0.6+mc1.19.jar
InventoryProfilesNext-fabric-1.19-1.6.0.jar
iris-mc1.19-1.2.5.jar
IronChests-1.7.2.jar
ItemBorders-1.19-fabric-1.1.6.jar
Jade-1.19-fabric-7.6.0.jar
journeymap-1.19-5.8.5rc2-fabric.jar
KronHUD-2.1.0.jar
lambdabettergrass-1.3.0+1.19.jar
lambdynamiclights-2.1.2+1.19.jar
lazydfu-0.1.3.jar
LegendaryTooltips-1.19-fabric-1.2.6.jar
levelz Compats.jar
levelz-1.3.6.jar
LibraryFerret-Fabric-1.19-2.0.0.jar
lists.txt
lithium-fabric-mc1.19-0.8.1.jar
logprot-fabric-1.19-1.6.jar.disabled
lootr-1.19-0.3.20.59.jar
malilib-fabric-1.19.0-0.13.0.jar
malum-quilt-b1.1.3.jar
mcda-2.1.1.jar
mcdar-1.5.0.jar
mcdw-4.4.0-1.19.jar
mcsa-2.4.0-1.19.jar
midnightlib-0.5.2.jar
minecraft-comes-alive-7.3.3+1.19-fabric.jar
minimotd-fabric-mc1.19-2.0.8.jar.disabled
mobs-attempt-parkour-0.3-1.19-pre1.jar
modmenu-4.0.4.jar
moremcmeta-1.19-3.0.4-fabric.jar
morevillagers-fabric-1.19-4.0.2.jar
MouseTweaks-fabric-mc1.19-2.22.jar
nameplate-1.0.3.jar
naturalist-fabric-2.0.1-1.19.jar
nicemod-1.1 - 1.19.jar
nyfsquiver-Fabric-0.5.0-1.19-rc2.jar
odysseyhud-fabric-1.19-v4.6.jar
Oh_The_Biomes_You'll_Go-fabric-1.19-2.0.0.7.jar
ostOverhaul-0.2.4-1.19 (1).jar
owo-lib-0.7.3+1.19.jar
Patchouli-1.19-74-FABRIC.jar
phosphor-fabric-mc1.19.x-0.8.1.jar
PresenceFootsteps-1.6.1.jar
Public GUI Announcement-fabric-1.19-4.0.0.0.jar
purpeille-3.0.0+fabric-1.19.jar
puzzle-1.3.4-1.19.jar
qfapi-2.0.0-beta.6_qsl-2.0.0-beta.14_fapi-0.57.0_mc-1.19.jar
reeses_sodium_options-1.4.3+mc1.19-build.43.jar
repurposed_structures_quilt-6.1.0+1.19.jar
rer-2.6.0.jar
RoughlyEnoughItems-9.1.517.jar
rpgdifficulty-1.2.0.jar
screenshot-to-clipboard-1.0.9-fabric.jar
sihywtcamd-1.7.1+1.19.jar
simplehats-fabric-1.19-0.1.2.jar
SimplyHouses-1.1.3-1.19-fabric.jar
SkyVillages-1.0.2.1-1.19.x-fabric.jar
smoothboot-fabric-1.19-1.7.1.jar
sodium-extra-0.4.5+mc1.19-build.43.jar
sodium-fabric-mc1.19-0.4.2+build.16.jar
spoornbountymobs-2.7.4-1.19.jar
spoornpacks-4.3.1-1.19.jar
stoneholm-1.4.4.jar
Subterrestrial-1.0.3.jar
TerraBlender-fabric-1.19-2.0.0.120.jar
terrarian-slimes-1.3-BETA+1.19.jar
The_Graveyard_1.10_(FABRIC)_for_1.19.jar
The_Graveyard_Biomes_1.2_(FABRIC)_for_1.19.jar
thonkutil-2.15.3+1.19.jar
thorium-1.3.1.jar
time-and-wind-ct-1.4.2+1.19.jar
totw_additions-1.2.3-1.19.x-fabric.jar
trashslot-fabric-1.19-12.0.1.jar
travelers-backpack-1.19-8.0.1.jar
trinkets-3.4.0.jar
trulytreasures-1.19-2.0.2.jar
twigs-fabric-2.0.0-beta.8+1.19.jar
VariableSpawnerHardness-1.19-1.2.2-fabric.jar
VillainousFishAI-fabric-1.19-1.1.jar
visuality-0.5.0.jar
voidtotem-fabric-1.19-2.0.0.jar
wardenloot-1.0.9-1.19.x.jar
waystones-fabric-1.19-11.0.1.jar
weaponmaster-multi-fabric-1-19.jar
whisperwoods-1.19-2.1.0-fabric.jar
wilderworld-2.3.0.jar
```

**Comment by quiqueck** [comment]
Thanks for creating an issue. I have just tested with the latest BetterEnd, and for me it looks like this:
<img width="386" alt="image" src="https://user-images.githubusercontent.com/280847/179375021-c733e442-16d9-4c2e-947e-24424d2b4188.png">

So I guess this is an external issue.

**Comment by quiqueck** [comment]
I think I found a workaround for this issue. Should be fixed in the next Release of BetterEnd/BCLib.

**Comment by quiqueck** [comment]
Should be fixed with 2.0.11
